src/Altinn.Common.PEP/Altinn.Common.PEP/Altinn.Common.PEP.csproj

```diff
@@ -3,14 +3,14 @@
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
     <OutputType>Library</OutputType>
-    <AssemblyVersion>3.0.0.0</AssemblyVersion>
-    <FileVersion>3.0.0.0</FileVersion>
+    <AssemblyVersion>4.0.0.0</AssemblyVersion>
```

Comment:
These two fields shouldn't bee necessary, no? If you remove them, don't you get the correct assembly version and file version?

Reply:
haven't tried it without them. Can check it out

Reply:
Yes, if not set, it defaults to 1.0.0

Reply:
I don't think this is correct, as I get the following output from msbuild:

![image](https://github.com/Altinn/altinn-authorization/assets/112334/41116099-4cca-4658-81a1-6a82f80f5713)

Reply:
Ah, it's cause you haven't set Version. You should replace <PackageVersion> with just <Version>. It sets all 3.